populus/defaults.py

```diff
@@ -0,0 +1,15 @@
+#
+# Config
+#
+GLOBAL_JSON_CONFIG_FILENAME = '.populus/populus.config.json'
+PROJECT_JSON_CONFIG_FILENAME = './project.json'
+PROJECT_JSON_CONFIG_DEFAULTS = 'project.config.defaults.json'
+PROJECT_JSON_CONFIG_SCHEMA = 'project.config.schema.json'
+
+#
+# Compile
+#
+DEFAULT_CONTRACTS_DIR = "./contracts/"
+DEFAULT_TESTS_DIR = "./tests/"
+BUILD_ASSET_DIR = "./build"
```

Comment:
Lets be consistent and either prefix all of these with `DEFAULT` or remove the prefix from the others.